lambdas/batch_process/batch_process_lambda.py

```diff
@@ -167,15 +167,15 @@ def batch_proc_once():
         except:
             print("Temporal parameter not found in batch proc. Defaulting to false.")
             temporal = False
+
         try:
-            if p.historical is True:
-                historical = True
+            processing_mode = p.processing_mode
+            if p.processing_mode == "historical":
                 temporal = True     # temporal is always true for historical processing
-            else:
-                historical = False
         except:
-            print("Historical parameter not found in batch proc. Defaulting to false.")
-            historical = False
+            print("processing_mode parameter not found in batch proc. Defaulting to forward.")
```

Comment:
do we want to allow execution in this case, or simply fail the lambda run? What is the scenario in which the parameter wouldn't be found rather than explicitly checked for?

Reply:
Not sure yet. I think that's a use-case we will refine as the actual operator gets hands-on experience on this feature.

You can create a batch_proc using a json file directly. If that json file doesn't contain processing_mode field, then the batch_proc object in ES will not have it. I can see us making this optional because this parameter into daac_data_subscriber query is also optional.

Reply:
maybe worth opening up a ticket so we know to revisit that use case later?